### ISSUE TITLE
cellTooltip not working correctly when cellFilter has just date format

### ISSUE THREAD

**Issue by venkat17** [angular-ui/ui-grid#5233]
Please make sure each of the following is true before submitting a new issue.  We are a small team with limited time and it is very difficult to manage issues if they aren't reported correctly.

- [x] This is a bug report, not a question on how to use the grid.  
Use [Stack Overflow](http://stackoverflow.com/questions/tagged/angular-ui-grid) or [Gitter](https://gitter.im/angular-ui/ui-grid) for questions.
- [x] You have searched the open issues to see if this bug has been filed before.  We don't want duplicate issues.
- [x] You have reviewed the tutorials and documentation to ensure you are using the grid correctly. [ui-grid.info](http://ui-grid.info/docs/#/tutorial)
- [x] You are using the latest version of the grid. Older versions are not patched or supported.
- [x] You have provided steps to recreate your bug.  A plunkr is even better.

If I give dateformat string and set cellTooltip: true, the tooltip is not displaying correctly. 
For example:
 `{ field: 'mixedDate', displayName: "Cust Date", cellFilter: 'date:"dd-MM-yyyy HH:mm:ss"', filterCellFiltered:true, cellTooltip: true, width: '30%'}`

As showing in the given plunker http://plnkr.co/edit/Olv74Y83jUaXQBvv8Mpg?p=preview

**Comment by angular-git-user** [comment]
you will basically have to add a function 
ex.
` cellTooltip: function(row, col) {
    return moment(row.entity.mixedDate); 
  }`

**Comment by kkankala** [comment]
ran into same issue. cellTooltip function will not get called if function is defiend for tooltip.

**Comment by venkat17** [comment]
Yes me too, the tool tip function calls are not working, if the column type is date and cellFilter date format is used.

**Comment by yogeshkumarpa** [comment]
Yes me too facing the same issue. Is this issue fixed in any newer version?

**Comment by cmorgant** [comment]
I ran into the same _issue_ that is not an issue actually...
I'd like to display a date in one format in the cell and in another one in the tooltip, unfortunately, this is not possible as long as you defined a cellFilter:

> Tooltips respect the cellFilter, so if you define a cellFilter it will also be used in the tooltip.

[Tutorial: 117 Tooltips](http://ui-grid.info/docs/#/tutorial/117_tooltips)

**Comment by Sabri55** [comment]
hi, when i use cellFilter, the cellTooltip don't work exemple :  
 {field: 'countryId', displayName: 'Location', enableCellEdit: true, editableCellTemplate:'uigrid/dropdownEditor',
cellFilter:"griddropdown:editDropdownOptionsArray:editDropdownIdLabel:editDropdownValueLabel:row.entity.country",editDropdownIdLabel: 'Key', editDropdownValueLabel: "Value"
cellTooltip: function (row, col) {
return 'Created : ' + row.entity.createDate + '  by: ' + row.entity.createUser   };
????????????????????????

**Comment by guille50** [comment]
This is not the final solution but at least works:

`{ field: 'mixedDate', displayName: "Cust Date", cellFilter: 'date:"dd-MM-yyyy HH:mm:ss"', filterCellFiltered:true, cellTooltip: true, width: '30%'}`

by 

`{ field: 'mixedDate', displayName: "Cust Date", cellFilter: 'date:\'dd-MM-yyyy HH:mm:ss\'', filterCellFiltered:true, cellTooltip: true, width: '30%'}`

Here is a plunkr with my working "solution"
http://plnkr.co/edit/mLk8FcfiyXIZTp1bKpAZ?p=preview

**Comment by freedomlang** [comment]
use single quote instead
```
cellFilter: "date:'dd-MM-yyyy HH:mm:ss'"
```

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
